pkg/op/session.go

```diff
@@ -98,5 +100,16 @@ func ValidateEndSessionPostLogoutRedirectURI(postLogoutRedirectURI string, clien
 			return nil
 		}
 	}
+	if globClient, ok := client.(HasRedirectGlobs); ok {
+		for _, uriGlob := range globClient.PostLogoutRedirectURIGlobs() {
+			matcher, err := glob.Compile(uriGlob)
```

Comment:
Are you aware of https://github.com/gobwas/glob#performance?

I understand that this is the only moment where you can actually compile the globs, as they come from storage and there is no way to store the compiled objects. However, I do would like to see some benchmarks to have an idea about performance impact using this library / approach. It might be just fine, but this sentence triggers me:

> This means, that compilation could take time, but strings matching is done faster,

Reply:
@muhlemmer I could add a LRU cache of compiled globs.  Or perhaps, I should add that to the glob library so that all users benefit?   Adding a cache is pretty easy but adds a dependency and a utility function.

Reply:
The nice LRU cache that is already indirectly part of the oidc package, github.com/hashicorp/golang-lru, has a nice generics-enhanced interface.   Are we ready to use generics in the `next` branch?